docs/services/jupyter-hub.md

```diff
@@ -1,145 +1,196 @@
-
-For more information on how to manage conda environments, refer to this [page](https://conda.io/docs/user-guide/tasks/manage-environments.html).
-
-From a terminal:
-```
-# Load a conda module
-module load conda
-conda activate base
-
-# set shell proxy variables to access external URL
-export http_proxy=http://proxy.alcf.anl.gov:3128
-export https_proxy=$http_proxy
-
-# create an environment name projectA
-conda create -y -n projectA
-
-# Activate conda environment
-conda activate projectA
-
-# Install required packages
-conda install -y jupyter nb_conda ipykernel mpi
-
-# Add environment to available kernel list
-python -m ipykernel install --user --name projectA
-
-# deactivate conda environment
-conda deactivate
-```
-
-Once the base environment is setup the user must add an `env` section to the `kernel.json` file, located in directory `${USER}/.local/share/jupyter/kernels/projecta`, defining the `CONDA_PREFIX` and `PATH` variables.  Currently, Polaris compute nodes access the internet through a proxy.  To configure the kernel to use the proxy add variables `http_proxy` and `https_proxy` to the `env` section.  This will allow users to install packages form within the notebook using `!conda` magic commands.  The following is a sample configuration:
-
-```
-{
- "argv": [
-  "/home/<user>/.conda/envs/projectA/bin/python",
-  "-m",
-  "ipykernel_launcher",
-  "-f",
-  "{connection_file}"
- ],
- "display_name": "projectA",
- "language": "python",
- "env": {
-    "CONDA_PREFIX":"/home/<user>/.conda/envs/projecta",
-    "PATH":"/home/<user>/.conda/envs/projecta/bin:${PATH}",
-    "http_proxy":"http://proxy.alcf.anl.gov:3128",
-    "https_proxy":"http://proxy.alcf.anl.gov:3128"
- },
- "metadata": {
-  "debugger": true
- }
-}
-```
-
-After completing these steps, you will see projectA kernel when you click new on the Jupyter Hub home page or when you use Kernel menu on a Jupyter notebook.
-
-
-## Accessing Project Folders:
-
-Jupyterhub file browser limits the user view files and directories within their home directory. To access project directory located outside of the user home directory a symbolic link to the directory mist be created within the user home directory. An example of this is:
+<figure markdown>
+  ![JupyterHub](files/Jupyter-0-login.png){ width="700" }
+  <figcaption>ALCF JupyterHub home page and sign-in screen</figcaption>
+</figure>
 
- If a user is connected to the Cooley instances and wants to access Theta project ABC, the user should execute the following commands to create the links:
+We describe below how to use JupyterHub on Polaris, ThetaGPU, Theta, and Cooley
+in more detail.
 
-```
-From terminal:
-ln -s /project/ABC ABC_project
-ln -s /lus/theta-fs0/projects/EFG EFG_project
+## Polaris
 
-From notebook:
-!ln -s /project/ABC ABC_project
-!ln -s /lus/theta-fs0/projects/EFG EFG_project
-```
+The Polaris JupyterHub instance runs on Polaris compute nodes through the job
```

Comment:
Apologies for being a bit pedantic, but it might be better to say:

> The Polaris JupyterHub server runs on a Polaris login node and launches individual users' notebook servers on Polaris compute nodes through the PBS job scheduler. 

Is this an accurate/more precise description @tjackson001?
See "Login nodes" section of https://docs.alcf.anl.gov/polaris/hardware-overview/machine-overview/

Reply:
I appreciate being thorough @felker. I wasn't sure if the users would be interested in where the JupyterHub server runs, but I made the following change:  [f3ae813](https://github.com/argonne-lcf/user-guides/pull/286/commits/f3ae81360ac6db537fb10ca4919e95a5530d712b) to add this information.